dominion/dominion_ui/src/components/Hero/HomeHero.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+import Hero, {
+  HeroTitle,
+  HeroDescription,
+  HeroTitleGrid,
+  HeroButton
+} from './Hero';
+
+import herobg from '../../assets/images/bg/hero_bg.png';
+import map from '../../assets/images/bg/hero_map.png';
+import smallscreenbackground from '../../assets/images/bg/smallscreen_background.png';
+
+import PortalChooser from '../Modal/PortalChooser';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1
+  },
+  heroContentGrid: {
+    flexGrow: 1,
+    backgroundImage: `url(${herobg})`,
+    backgroundRepeat: 'no-repeat',
+    backgroundPosition: 'center',
+    backgroundSize: 'auto',
+    width: '100%',
+    height: '100%',
+    [theme.breakpoints.down('sm')]: {
+      backgroundImage: `url(${smallscreenbackground})`,
+      backgroundPosition: 'right top'
+    }
+  },
+  verticalAlignText: {
+    color: 'white',
+    writingMode: 'vertical-lr',
+    textOrientation: ' sideways-right',
+    marginLeft: '10px'
+  },
+  mapSection: {
+    color: 'white',
+    //    paddingLeft: '5rem',
+    [theme.breakpoints.down('sm')]: {
+      display: 'none'
+    }
+  },
+  modalContent: {
+    margin: 'auto',
+    top: theme.spacing.unit * 10,
+    height: 'auto'
+  }
+});
+
+function HomeHero({ classes }) {
+  return (
+    <Hero>
+      <Grid
+        container
+        direction="row"
+        justify="space-between"
+        alignItems="center"
+        className={classes.heroContentGrid}
+      >
+        <HeroTitleGrid>
+          <HeroTitle>Discover the stories behind the data.</HeroTitle>
+
+          <HeroDescription>
+            Dominion makes data available to help add context and authority to
+            public discourse and policy-making on vital issues of land
+            ownership.
+          </HeroDescription>
+
+          <PortalChooser
+            activator={({ handleToggle }) => (
+              <HeroButton onClick={handleToggle}>Select a Country</HeroButton>
+            )}
+          />
```

Comment:
The modal portal chooser, from `select a country` does not change the arrow down/up on the nav. Is `handleToggle` passed on HeroButton initialised on HomeHero?

Reply:
I'll fix that, i might've missed that part.